guides/ruby-gems/gemfury.md

```diff
@@ -0,0 +1,50 @@
+# Gemfury usage
+
+Alternative way to use private gems on multiple projects and add them to dependencies is publishing on hosted repository [gemfury](https://manage.fury.io/dashboard/dvelp).
```

Comment:
[Gemfury](https://manage.fury.io/dashboard/dvelp) is a private gem server that we use to easily manage access to private gems.